src/routes/console/project-[project]/settings/updateName.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+    import { invalidate } from '$app/navigation';
+    import { base } from '$app/paths';
+    import { page } from '$app/stores';
+    import { Submit, trackError, trackEvent } from '$lib/actions/analytics';
+    import { CardGrid, CopyInput, Heading } from '$lib/components';
+    import { Dependencies } from '$lib/constants';
+    import { Button, Form, FormList, InputText } from '$lib/elements/forms';
+    import { addNotification } from '$lib/stores/notifications';
+    import { sdk } from '$lib/stores/sdk';
+    import { onMount } from 'svelte';
+    import { project } from '../store';
+
+    const endpoint = sdk.forConsole.client.config.endpoint;
+    const projectId = $page.params.project;
+
+    let name: string = null;
+
+    onMount(() => {
+        name = $project.name;
+    });
+
+    async function updateName() {
+        try {
+            await sdk.forConsole.projects.update($project.$id, name);
+            await invalidate(Dependencies.PROJECT);
+            addNotification({
+                type: 'success',
+                message: 'Project name has been updated'
+            });
+            trackEvent(Submit.ProjectUpdateName);
+        } catch (error) {
+            addNotification({
+                type: 'error',
+                message: error.message
+            });
+            trackError(error, Submit.ProjectUpdateName);
+        }
+    }
+</script>
+
+<Form onSubmit={updateName}>
+    <CardGrid>
+        <Heading tag="h6" size="7">API Credentials</Heading>
```

Comment:
We probably shouldn't call this Credentials since that makes it sound like it's sensitive.

Reply:
cc @gewenyu99

Reply:
What does this do?